protobuf-maven-plugin/src/main/java/io/github/ascopes/protobufmavenplugin/mojo/AbstractGenerateMojo.java

```diff
@@ -242,6 +245,38 @@ public AbstractGenerateMojo() {
   @Parameter(defaultValue = DEFAULT_FALSE)
   boolean embedSourcesInClassOutputs;
 
+  /**
+   * Source paths to exclude from compilation.
+   *
+   * <p>This can be used to limit what is compiled by {@code protoc}.
+   *
+   * <p>Each entry is treated as a glob pattern, and is applied to the path of each discovered
+   * compilation candidate file, relative to the {@code sourceDirectory} or {@code sourceDependency}
+   * that provides them.
+   *
+   * <p>If a file matches <strong>any</strong> of these patterns, it is automatically excluded.
+   *
+   * <p>If not provided, then the default is to not exclude anything.
+   *
+   * <p>For example, if you wanted to not compile files named {@code user.proto},
+   * {@code message.proto}, or {@code service.proto}, you could use the following
+   * configuration.
+   *
+   * <pre>{@code
+   * <excludes>
+   *   <exclude>**\/user.proto</exclude>
```

Comment:
- [ ] Check these forward-slash escapes render correctly in the generated Maven plugin site.